superset/charts/commands/delete.py

```diff
@@ -45,8 +45,13 @@ def __init__(self, model_id: int):
 
     def run(self) -> Model:
         self.validate()
+        self._model = cast(Slice, self._model)
```

Comment:
is this necessary? The doc said `This returns the value unchanged.` but only good for type checker

Reply:
It's needed to make the type checker happy.

On `__init__` `self._model` will be set to `None`. When we call `self.validate()` it will either set `self._model` to a `Slice` or raise an exception, so here we're telling the type checker that we know `self._model` is not of type `Slice`.